### ISSUE TITLE
How to use sphinxbase with Xcode

### ISSUE THREAD

**Issue by Shinehah-Gnolaum** [cmusphinx/sphinxbase#68]
At [these instructions in your readme for Xcode](https://github.com/cmusphinx/sphinxbase#xcode-installation-for-iphone) you say:

> Sphinxbase uses the standard unix autogen system, you can build sphinxbase with automake. You just need to pass correct configure arguments, set compiler path, set sysroot and other options. After you build the code you need to import dylib file into your project and you also need to configure includes for your project to find sphinxbase headers.

Would you give more detail for those instructions? I am unfamiliar with what is said there.

**Comment by nshmyrev** [comment]
The next line says

```For details see http://github.com/cmusphinx/pocketsphinx-ios-demo```

I also already advised you to use tlsphinx.

**Comment by Shinehah-Gnolaum** [comment]
@nshmyrev Ok. I did take both those two routes and ran into an error in each one. I backtracked here in hopes that I would be able to proceed with the pocketsphinx. It looks like the error I got with it would be solved if I knew more how to configure pocketsphinx. I actually tried to run build_iphone.sh from the pocketsphinx-ios-demo repository on sphinxbase first, before I tried it on pocketsphinx, that's why I posted the error here instead of at the pocketsphinx repository. I'm getting the same error running the build_sphinx.sh on pocketsphinx. Should I post an issue at the pocketsphinx repository?
